packages/reusable-blocks/src/store/controls.js

```diff
@@ -10,6 +10,11 @@ import {
 import { createRegistryControl } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { store as reusableBlocksStore } from './index.js';
```

Comment:
This is surprising to see a cyclic dependency between `store/index.js` and `store/controls.js`. Would it be possible to access somehow `dispatch` from the current store instead? /cc @adamziel

Reply:
@gziolo The control needs to dispatch data on that store so it's not easily avoidable. I think the only way of avoiding circular dependency would be not refactoring this file and just leaving "core/reusable-blocks" string as it was before. This is going to be a general theme in plenty of registry actions and selectors too - we should address all those cases in the same way and either accept the circular dependency (it shouldn't break anything in this case) or use string to refer to the store from the same package.

Reply:
True, reusing the constant sounds like a good way of handling this particular case 👍